lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -288,7 +288,7 @@ def log_write_failure(request, error)
     end
 
     def fetch_metadata(metadata_path)
-      open('http://metadata/computeMetadata/v1/' + metadata_path,
+      open('http://169.254.169.254/computeMetadata/v1/' + metadata_path,
```

Comment:
Would it be useful to leave a link to docs here?
https://cloud.google.com/compute/docs/metadata